### PR TITLE
product_destination_json

### PR DESCRIPTION
**Pull request by 006080**
product.json file added

### CODE REVIEW

**Comment by BajalanIman** [comment]
Your work is great. However, I believe it is better to put your product.json file inside a folder (like data or whatever you prefer).

**Comment by 006080** [comment]
Ok. No problem

**Comment by BajalanIman** [comment]
I'm not sure, but maybe the name of the folder should be lowercase (data rather than Data).
https://github.com/felixge/node-style-guide

**Comment by BajalanIman** [comment]
It seems @dvnishshanka also has done this. I'm just wondering why you both have done this ticket, as his branch contains the products.json file.
image

**Comment by dvnishshanka** [comment]
@BajalanIman @006080 Hi, When I was doing the ticket the products.json was not merged. So I put the data which we already had in App to a products.json file. So we still need the proper products.json file.

File: server/Data/product.json [hunk 1]
```diff
@@ -0,0 +1,162 @@
+[
+  {
+    "id": 1,
+    "name": "Santorini",
+    "location": "Greece",
+    "description": "A stunning island known for its whitewashed houses and incredible sunsets.",
+    "price": "$2000",
```
Comment (matus-vacula) [inline server/Data/product.json:7]: Please remove price attribute from each record

**Comment by matus-vacula** [comment]
Please rename the file to server/src/data/destinations.json

**Pull request closed without merging**